### DISCUSSION TITLE
Restart same sensor in patched LibreLink App (first session started without account)

### DISCUSSION THREAD

**Discussion by zwidow** [maheini/FreeStyle-Libre-3-patch#30] [Q&A] [answered]
Hi!
Someone know if with the patched LibreLink-App is possible to restart the same sensor?
I fall in the "NOT UPDATED" instruction page... 😡[https://github.com/.../instructions/en/instructions.md](https://github.com/maheini/FreeStyle-Libre-3-patch/blob/main/Juggluco-solution/instructions/en/instructions.md?fbclid=IwAR0U8f6po2BQbpcJxWrm9MCQs_0ajUYvQI1G3QUTINxs3ET7vs9woSrWSMk)
that made me start the sensor without an active account.. now I can't switch data and sensor to Juggluco unless I start a new sensor in the patched app with an active account.....

**Comment by maheini** [comment]
If you didn't started your sensor whilst logged in, there's no chance to restart the sensor again.

The patched libre3 app does allow you to re-enable sensors which are previously enabled with an active account.

What do you mean by "NOT UPDATED" instruction page? Are you trying to switch from Patched app -> Juggluco to Juggluco direct solution?

If so, just chill and wait until the sensor expires. There are not huge benefits. Btw. there's a instruction for this solution in this repo as well (currently german only, but you can use google translate):  https://github.com/maheini/FreeStyle-Libre-3-patch/blob/main/Juggluco-solution/juggluco-direct-instructions/de/instructions.md

English version will follow within the next month.

**Reply by zwidow** [reply]
I follow the old instruction but have no data on Juggluco, the connection between Patchedapp and Juggluco seems ok but still no data.
So I start search and find the "new" solution but I can't switch (due to account problem) and I can't see data in Juggluco.
![image](https://user-images.githubusercontent.com/80387707/207149113-d9ee5be2-3a04-4eb0-b6fe-01e9b17c157b.png)
![image](https://user-images.githubusercontent.com/80387707/207149137-daef5b54-5ca6-44dc-b760-f078835fa5fd.png)

**Comment by maheini** [comment]
Hang in there - it will be working. Everything is tested, even though it's a little bit more complex than the direct solution.

In general, your screenshots do look good. Please check those settings: Have you set the port 7117 at the top left? Everything else as well?

![grafik](https://user-images.githubusercontent.com/65506676/207149662-066b8ee4-872e-44d2-b4d8-32884ade0733.png)

If it is still not working, try to restart your phone and wait some time after. Try to press "Sync" and "Syncronize" withing Libre/Juggluco.

And if there is still no reading visible, could you please make a screenshot of the detailed setting within Juggluco and Libre? Thanks.

**Comment by zwidow** [comment]
![image](https://user-images.githubusercontent.com/80387707/207156129-96cb9e14-4bfa-4b08-99fb-6278e97eb264.png)
![image](https://user-images.githubusercontent.com/80387707/207156199-3ac57722-ec23-4267-a64a-d6facdbb6afa.png)
I deleted and recreate the connection in Juggluco, somthing change phone vibrating and a pop up appear with a blood sugar value.
But then nothing change, and still no data

**Comment by maheini** [comment]
Those settings do look good as well. 

Some questions to solve the problem:

- So you received at least one reading within Jugglugo? Or just some empty message?
- I guess the patched libre app does receive readings, right?
- Are you using a VPN or proxy server on your phone?

If all those questions seem to be fine, then you might try the older Juggluco version: https://github.com/maheini/FreeStyle-Libre-3-patch/raw/400480da4bda8c57ff3622b3cb4cacd189cfcab5/Juggluco-solution/versions/latest/Juggluco.apk

**Comment by zwidow** [comment]
Yes I have data in the patched libre app.
I recived a pop up with a blood glucose number then nothing.
Is a spare phone connected in wi-fi no VPN or proxy
I've tryed the old version but nothing change.
I have some strange things, always have a notify in Juggluco (also the new version) that never disappear and don't know what it means.
![image](https://user-images.githubusercontent.com/80387707/207169682-b2d395c7-572d-40f4-94be-dea33b577a2a.png)

The old version every time I open Juggluco ask me if I want to start without sensor or just open the app maybe is the standard but seems strange.

**Reply by maheini** [reply]
Hmmm... kind of hard to debug. I never had a notification which isn't visible inside Juggluco. The sensor-start check is normal for the old Juggluco app.

Here is a troubleshooting guide: [link](https://github.com/maheini/FreeStyle-Libre-3-patch/blob/main/Juggluco-solution/instructions/en/instructions.md#troubleshooting-libre3---juggluco-connection)

If you haven't done yet, try to disable battery Optimisations for Juggluco, to keep it alive in the background.

I suspect the problem within the synchronization between Juggluco and Libre patch.

**Reply by maheini** [reply]
You can try to force-stop Juggluco. This should disable the notification at least for a moment and maybe even fix the problem

**Comment by zwidow** [comment]
The "Resend data" from libre app solve the issue in the old version now I have the data. Still have the notification also, but for now is ok.
I also have data in xdrip

**Comment by maheini** [comment]
Hurray! 😄 

Glad to hear about that.

**Comment by maheini** [comment, marked as answer]
To sum it all up (for people with a similar problem):

- Try installing the previous Juggluco version: [download link](https://github.com/maheini/FreeStyle-Libre-3-patch/blob/main/Juggluco-solution/juggluco-direct-instructions/de/instructions.md)
- Set everything up (exactly as described within the instructions)
- Try pressing Resend data & Resync buttons within Juggluco & patched Libre 3 app
- Try restarting your phone.
